sbi/inference/base.py

```diff
@@ -105,7 +105,7 @@ def __init__(
                 maps to data x at once. If None, we simulate all parameter sets at the
                 same time. If >= 1, the simulator has to process data of shape
                 (simulation_batch_size, parameter_dimension).
-            device: torch device on which to compute, e.g. 'cuda', 'cpu'.
+            device: torch device on which to compute, e.g. `gpu` or `cpu`.
```

Comment:
For all strings, I have so far just dropped any `` or '', since it is already indicated in the type that it's a string

Reply:
Aside: in snle_a you write it without backticks as well :)